indigo_app/views/works.py

```diff
@@ -1535,3 +1536,56 @@ def get_context_data(self, **kwargs):
         )
         context_data["prefix"] = prefix
         return context_data
+
+
+class WorkFormConsolidationView(WorkViewBase, TemplateView):
+    template_name = "indigo_api/_work_form_consolidations_form.html"
+
+    def post(self, request, *args, **kwargs):
+        return self.get(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        formset = ConsolidationsBaseFormset(self.request.POST,
+                                            user=self.request.user,
+                                            work=self.work,
+                                            prefix="consolidations",
+                                            form_kwargs={"work":self.work,
+                                                         "user": self.request.user})
+        initial = []
+        if formset.is_valid():
+            for form in formset.forms:
+                delete = form.cleaned_data.get("DELETE")
+                if delete and not form.cleaned_data.get("id"):
+                    continue
+                initial.append({
+                    "date": form.cleaned_data["date"],
+                    "id": form.cleaned_data["id"],
+                    "DELETE": form.cleaned_data["DELETE"],
+                })
+
+            if self.request.POST.get("consolidation") == "add":
+                date = datetime.datetime.today()
+                # if (
+                #     self.place.place_settings.is_consolidation and
+                #     self.place.place_settings.as_at_date and
+                #     self.work.principal and
+                #     self.work.publication_date and
+                #     self.work.publication_date > self.place.place_settings.as_at_date
+                # ):
+                #
+                #     date = self.place.place_settings.as_at_date
```

Comment:
@goose-life How do I determine the correct place settings to use for a place? it seems we  do not have a one to one relation for a place and place settings